### PR TITLE
Zend render caching

### PR DESCRIPTION
**Pull request by ahilles107**
- use templates service in zend controllers rendering
- improve vector in service

### CODE REVIEW

File: newscoop/library/Newscoop/SmartyView.php [hunk 1]
```diff
@@ -11,30 +11,28 @@
...
-        foreach ($this->getVars() as $key => $val) {
-            $this->smarty->assign($key, $val);
-        }
+        $container = \Zend_Registry::get('container');
+        $templatesService = $container->getService('newscoop.templates.service');
+        $request = $container->getService('request');
+        $language = $container->get('em')->getRepository('Newscoop\Entity\Language')->findOneByCode($request->getLocale());
 
-        $this->smarty->assign('view', $this);
-        $this->smarty->assign('gimme', $this->smarty->context());
```
Comment (takeit) [inline newscoop/library/Newscoop/SmartyView.php:34]: won't that break anything ? wasn't it used anywhere ?

Comment (ahilles107) [inline newscoop/library/Newscoop/SmartyView.php:34]: `gimme` is set already in ThemesService (in construct) and `view` is set manually few lines bellow.

Comment (takeit) [inline newscoop/library/Newscoop/SmartyView.php:34]: you mean `TemplatesService`, ok then it looks 👍

Comment (ahilles107) [inline newscoop/library/Newscoop/SmartyView.php:34]: Yes `TemplatesService`.

**Pull request merged**